django-data/image/submissions/templates/submissions/submission_detail.html

```diff
@@ -187,6 +187,18 @@ <h1>Submission Detail for <em>{{ submission.title }}</em></h1>
       }
   }
 
+  function construct_validation_message(modelType, message, unknown, issues) {
+      modelLink = `validation_summary/${modelType}/`;
```

Comment:
I think that django `url` template tags works also here, please check

Reply:
Tried several solutions including JS string interpolation, JS replace method, constructing string dynamically ("type=" + modelType) and it doesn't work. I understand that url schema might change in the future but we have a lot of test and we should catch any bug that would arise because of that, so let's leave it as it is by now.